db-client-java/src/test/java/com/eventstore/dbclient/DeletePersistentSubscriptionTests.java

```diff
@@ -15,4 +11,13 @@ public void testDeletePersistentSub() throws Throwable {
         client.delete("aStream", "aGroupUpd")
                 .get();
     }
+
+    @Test
+    public void testDeletePersistentSubToAll() throws Throwable {
+        client.createToAll("aGroupUpd")
+                .get();
+
+        client.deleteToAll("aGroupUpd")
+                .get();
+    }
```

Comment:
**CONCERN:** Shouldn't we have some assertion that verifies if the operations succeeded? We can have the false-positive test results if delete silently fails.

Reply:
This is how the current tests are written, currently the `List`/`Get` operations aren't implemented yet for persistent subscriptions. I would suggest to implement this in separate PR and modify/add the tests.

Reply:
I'm good with adding them in the separate PR, but it'd be good to create an issue for that to not forget about it.

Reply:
Done https://github.com/EventStore/EventStoreDB-Client-Java/issues/87